jerry-core/vm/opcodes.c

```diff
@@ -756,6 +757,7 @@ opfunc_resume_executable_object (vm_executable_object_t *executable_object_p, /*
 #endif /* JERRY_BUILTIN_REALMS */
 
   ecma_value_t result = vm_execute (&executable_object_p->frame_ctx);
+  ecma_deref_if_object (executable_object_p->frame_ctx.this_binding);
```

Comment:
As I see it, the modification is half-correct to one of the reviews on the original PR ( https://github.com/jerryscript-project/jerryscript/pull/4966#discussion_r784953708
 ) , but it ignores the original question and its corresponding solution, where the questioned part was connected with the release. 
 The condition that contains the check that whether executable_object_p->frame_ctx.call_operation is VM_EXEC_RETURN or not,  is still placed  after this deref call.

Reply:
Sorry. I missed the deref part. Hopefully it is now correct.